templates/three-template/script.js

```diff
@@ -0,0 +1,96 @@
+import {
+  WebGLRenderer, 
+  PerspectiveCamera,    
+  Scene, 
+  Mesh, 
+  BoxGeometry,
+  MeshNormalMaterial,
+  Clock 
+} from "./three.module.js";
+
+import { OrbitControls } from "./OrbitControls.js";
+
+// If you want to create OBJKT's with different seeds, 
+// you can access the creator and viewer wallet ids. 
+// This values will only be injected once the piece 
+// has been minted they will not work locally.
+// If the user is not sync, the viewer comes in as false.
+const creator = new URLSearchParams(window.location.search).get('creator');
+const viewer = new URLSearchParams(window.location.search).get('viewer');
+
+console.log('NFT created by', creator);
+console.log('NFT viewed by', viewer);
+
+class Sketch {
+  constructor() {
+    this.renderer = new WebGLRenderer({ 
+      antialias: true, 
+      alpha: true, 
+    });
+    this.renderer.setPixelRatio(Math.min(window.devicePixelRatio, 1.5));
```

Comment:
I normally use 2 instead of 1.5. Never tried it like this! have you ever tested it on a retina display?

Reply:
I can change it, haven't tested it either, grabbed the number from the reference repo.